priv/data.yml

```diff
@@ -1,2 +1,2 @@
-- Lorem ipsum
-- Dolor sit amet
+- "ae meu princesos e principas, tudo bem? Vocês sabiam que em 18 de dezembro de 1954 foi fundado no interio do Rio Grande do Sul o município de Não-Me-Toque? Este que tem apenas 17mil habitantes e um pib de aprox. R$ 462mil, aqui na :ptec:  nós ja fechamos contratos maiores, faturamos mais que uma cidade toda! Não-Me-Toque também me lembra do classico noventista You Cant Touch This do  MC Hammer que claramente foi inspirado pela pequena cidade rio grandense. Sabe oque é 18 também mas não é para dezembro? O review do meu PR, 18 linhas só po :gato_de_botas:"
+- "Meus nobres, voces sabiam que a 4000 anos atras, a humanidade na Mesopotamia (do Grego, terra entre Rios) desenvolvia com simbolos cuneiformes a matemática de base 60 ela foi usada principalmente para representar tempo e expecula-se que um dos principais motivos era a possibilidade de contar até 12 com apenas uma mão (pode tentar agora!! Use seu dedão para contar cada gomo dos seus outros 4 dedos). Sabe o que mais é 4000?? O ID do meu novo PR no webapp que está aguardando o seu review ♥"
```

Comment:
```suggestion
- "Meus nobres, voces sabiam que há 4000 anos atrás a humanidade na Mesopotâmia (do Grego, terra entre Rios) desenvolvia com símbolos cuneiformes a matemática de base 60? Ela foi usada principalmente para representar tempo e especula-se que um dos principais motivos era a possibilidade de contar até 12 com apenas uma mão (Pode tentar agora! Use seu dedão para contar cada gomo dos seus outros 4 dedos). Sabe o que mais é 4000? O ID do meu novo PR no webapp que está aguardando o seu review ♥"
```

Reply:
The portuguese of this @CaiqueMitsuoka sucks

Reply:
@CaiqueMitsuoka help us to change this last part of the phrase:
```Sabe o que mais é 4000? O ID do meu novo PR no webapp que está aguardando o seu review ♥```
It needs to be more generic

Reply:
WDYT?
```
Especula-se também que esse serumaninho aqui decendente desses seres humanos da mesopotâmia também ficaria muito feliz com esse seu approve nesse PR ❤️ 
```